OmarEspinosaCastro/30-Agosto/ResultHashTables.java

```diff
@@ -0,0 +1,25 @@
+
+package collection;
+
+public class ResultHashTables {
+    static boolean checkMagazine(String[] magazine, String[] note) {
+        HashMap<String, Integer> magazineWords = new HashMap<String, Integer>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices